{"name": "Авторизация под пользователем, который есть в системе", "status": "passed", "description": "При авторизация под пользователем, который есть в системе, происходит успешная авторизация", "start": 1744200580903, "stop": 1744200581367, "uuid": "51e365ea-abe7-426b-8d43-e92c138eb463", "historyId": "c1ee95458348aabac0acb25665322199", "testCaseId": "c1ee95458348aabac0acb25665322199", "fullName": "tests.test_login_user.Testlogin#test_login_user", "labels": [{"name": "suite", "value": "Авторизация пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "Testlogin"}, {"name": "host", "value": "DESKTOP-GVJUSQ6"}, {"name": "thread", "value": "6996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}